vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj

```diff
@@ -31,7 +31,7 @@
     <ProjectGuid>{88BAB0FA-E1AC-4DA7-A30C-F91702A8EADB}</ProjectGuid>
     <ProjectName>React.Windows.Desktop.DLL</ProjectName>
     <TargetName>react-native-win32</TargetName>
-    <USE_V8>true</USE_V8>
+    <UseV8>true</UseV8>
```

Comment:
The casing changes related to USE_V8 and USE_HERMES was because we expose the props to the user project now, juxtaposed with other MSBuild props. Changed these MSBuild style for consistency,

Reply:
Ideally all our MSBuild variables must have our project prefix. E.g. RN, RNW, React, or ReactNative. Since all MSBuild properties are global we will have less probability to conflict with settings defined in other projects. E.g. all C++/WinRT properties have the CppWinRT prefix which makes it easier to recognize and differentiate from other project types.

---
In reply to: [513096203](https://github.com/microsoft/react-native-windows/pull/6350#discussion_r513096203) [](ancestors = 513096203)

Reply:
I agree with this. Doing this consistently would be a pretty large change though, so it likely doesn't make sense for this PR.